Source/ChocolateyGui/ViewModels/SettingsViewModel.cs

```diff
@@ -251,7 +234,10 @@ public void Back()
             WireUpConfig();
 
             var features = await _packageService.GetFeatures();
-            Features.AddRange(features);
+            foreach (var feature in features)
+            {
+                Features.Add(feature);
+            }
```

Comment:
Actually could be optimized further. I don't what library I pulled in had it, but ObservableCollection (which backs Features, Settings, and Sources on this page) doesn't have an `AddRange`, so I had to fall back to good old fashion add.